setup.py

```diff
@@ -136,6 +136,8 @@ def _parse_requirements_file(file_path):
     "openpifpaf==0.13.6",
     "opencv-python<=4.6.0.66",
 ]
+_yolov8_integration_deps = _yolo_integration_deps + ["ultralytics"]
```

Comment:
Should we pin version here? Are the things we overrode in the classes in danger of being broken by a breaking change in ultralytics repo?

Reply:
is ultralytics versioned? They have no tags, no branches, no releases

Reply:
Yeah you have to have a version to publish to pypi, they just aren't tagging them on their github repo

https://pypi.org/project/ultralytics/#history

Reply:
I cannot use the same version as you did in sparseml, because it does not have the fix for square images. Will use the most recent version.